main.go

```diff
@@ -272,7 +273,6 @@ func initFlags() {
 
 	if config.Format && config.FormatAs == "" {
 		config.FormatAs = "markdown"
-		config.FormatText = defaultConfig().FormatText
```

Comment:
i think this was needed

Reply:
Yes, you are right. This was added to handle cases where the `FormaText` can't be initialized from the `settings.` Without this line, the process fails. Nonetheless, it introduces a bug that doesn't allow the use of custom format text entries.

In the last commit, I added the line to a different text, making it work as expected.

I've tested it with these `settings`:

```yaml
format-text:
  code: |
    Format the response as code within appropriate code fences, specifying the correct language. Refrain from adding comments or explanations
    unless explicitly requested. Adhere to best practices for the given language, ensuring the code is well-structured and aesthetically
    pleasing.
```

If you try to run `mods` with the `--format --format-as code` options in the current build, it doesn't run. But on this PR, it does.